ajax/libs/venn.js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "venn.js",
+  "filename": "venn.min.js",
+  "description": "A javascript library for laying out area proportional venn and euler diagrams",
+  "homepage": "https://github.com/benfred/venn.js/",
+  "author": "Ben Frederickson",
+  "keywords": [
+    "venn-diagram",
+    "javascript",
+    "d3",
+    "euler-diagram"
```

Comment:
Let's add more keywords, such as `venn` and `euler` https://github.com/benfred/venn.js/blob/master/bower.json#L11-L13
Thanks